hphp/runtime/vm/jit/relocation-arm.h

```diff
@@ -0,0 +1,45 @@
+/*
+   +----------------------------------------------------------------------+
+   | HipHop for PHP                                                       |
+   +----------------------------------------------------------------------+
+   | Copyright (c) 2010-2016 Facebook, Inc. (http://www.facebook.com)     |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | http://www.php.net/license/3_01.txt                                  |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+*/
+#ifndef incl_HPHP_CODE_RELOCATION_ARM_H_
```

Comment:
Nix the `CODE_`—include guards should be just `incl_HPHP_FILENAME_ALL_CAPS_WITH_UNDERSCORES_H_`.

(Also, maybe a newline above it.)

This comment also applies for all the other relocation headers.